### DISCUSSION TITLE
unexpected keyword argument 'prompt_embeds'

### DISCUSSION THREAD

**Discussion by goedelesaar** [azuritecoin/OnnxDiffusersUI#129] [Q&A]
Title shows me the error that im getting. 

The problem is recreatable whenever i have started the webUI, entered a prompt, and clicked generate

A bit more info:

```
File "D:\Stable_diffusion\stable_diff\virtualenv\lib\site-packages\diffusers\pipelines\stable_diffusion\pipeline_onnx_stable_diffusion.py", line 365, in __call__
    prompt_embeds = self._encode_prompt(
TypeError: _encode_prompt() got an unexpected keyword argument 'prompt_embeds'
```

Line 365 (+ more)

```
        # here `guidance_scale` is defined analog to the guidance weight `w` of equation (2)
        # of the Imagen paper: https://arxiv.org/pdf/2205.11487.pdf . `guidance_scale = 1`
        # corresponds to doing no classifier free guidance.
        do_classifier_free_guidance = guidance_scale > 1.0

365>        prompt_embeds = self._encode_prompt(
            prompt,
            num_images_per_prompt,
            do_classifier_free_guidance,
            negative_prompt,
            prompt_embeds=prompt_embeds,
            negative_prompt_embeds=negative_prompt_embeds,
        )
```

using the CLI's ```python txt2img_onnx.py --prompt="tire swing hanging from a tree" --height=512 --width=512``` works, so it seems to be only the UI that's acting this way

Also this is my first time posting on github, pelase be nice! ;)

**Comment by LarkTelby** [comment]
Having the same problem. Did you find a solution?

**Reply by goedelesaar** [reply]
none yet unfortunately...

**Comment by uchuusen** [comment]
I've created a PR with a workaround for this issue. https://github.com/azuritecoin/OnnxDiffusersUI/pull/131
It should solve the problem, I hope.
